index.html

```diff
@@ -51,6 +51,9 @@
 <li>Soulspace Salon &amp; Yoga Studio in Orchard Park</li>
 <li>Personal Studio in Lackawanna</li>
 </ul>
+        <ul>
+          <li> Private sessions and classes available upon request. To schedule a private session please email me at AKHigginsYoga@gmail.com</li>
+        </ul>
```

Comment:
The syntax checks out, but maybe a paragraph <p></p> would be a better idea here? This syntax is for a list.